### PR TITLE
Update Build team permissions

### PR DESCRIPTION
**Pull request by mmorhun**
Requested permissions allow Build Team better troubleshoot issues with maintained services.

### CODE REVIEW

**Review by arewm — APPROVED** [review]

**Review by sbose78 — APPROVED** [review]
Looks good!

**Comment by manish-jangra** [comment]
Can we use groups in bindings instead of individual users?

**Comment by amfred** [comment]
> Can we use groups in bindings instead of individual users?

I second this.  Can you define a group or groups in infra-deployments and use the Group Sync Operator to synchronize the group with a Github Team?  We can synchronize Github Teams from infra-deployments to the clusters.  It's just icky technical debt to have individuals' usernames in the infra-deployments code.

**Comment by lkolacek** [comment]
Changes look OK to me, but if there's a way to define groups instead of individual users, it would make the maintenance easier.

**Comment by mmorhun** [comment]
@manish-jangra @amfred I've updated PR, please review again.
Also, I'd like to ask you to check the group name as I cannot list groups to see the respective name.

File: components/image-controller/base/rbac/image-controller-admin.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - delete
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: image-controller-admins
+  namespace: image-controller
+subjects:
+  - kind: User
```
[the remark this answers is not in the document]
Comment (mmorhun) [reply components/image-controller/base/rbac/image-controller-admin.yaml]: ok

File: components/build-service/base/rbac/build-admin.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - delete
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: build-admins
+  namespace: build-service
+subjects:
+  - kind: User
```
[the remark this answers is not in the document]
Comment (mmorhun) [reply components/build-service/base/rbac/build-admin.yaml]: ok

**Review by amfred — APPROVED** [review]

**Review by lkolacek — APPROVED** [review]
Looks great Mykola! 👍 Thank you

**Pull request merged**